apps/web/src/app/support/_utils/config.ts

```diff
@@ -0,0 +1,6 @@
+export const CURRENCY = 'usd';
+// Set your amount limits: Use float for decimal currencies and
+// Integer for zero-decimal currencies: https://stripe.com/docs/currencies#zero-decimal.
+export const MIN_AMOUNT = 10.0;
+export const MAX_AMOUNT = 5000.0;
+export const AMOUNT_STEP = 5.0;
```

Comment:
:+1: lg, might drop the decimal `.0` on ea but unimportant

Reply:
ah damn these are actually dead values that were pasta'd. will remove!